recipe/meta.yaml

```diff
@@ -11,9 +11,11 @@ source:
     fn: {{ name }}-{{ version }}.tar.gz
     url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
     {{ hash_type }}: {{ hash_val }}
+    patches:
+        0001-Remove-PySpark3-kernel.patch
 
 build:
-    number: 1000
+    number: 1001
```

Comment:
Let's try something a little experimental here. Make the version by `'1!{{ version }}'`, and change the build number to `0`.